### PR TITLE
Refreshed APIs, added helper apps to dump APIs, DontRestrictGroovy flag

### PR DESCRIPTION
**Pull request by biocomp**

### CODE REVIEW

File: src/main/groovy/me/biocomp/hubitat_ci/api/common_api/WebSocket.groovy [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package me.biocomp.hubitat_ci.api.common_api
+
+// hubitat.helper.interfaces.WebSocket
+interface WebSocket
+{
+    abstract void close()
+    abstract void connect(String a)
+    abstract void connect(Map a, String b)
+    abstract void sendMessage(String a)
+}
```
Comment (cmorse) [inline src/main/groovy/me/biocomp/hubitat_ci/api/common_api/WebSocket.groovy]: Sweet! I'm going to need this interface.

**Comment by biocomp** [comment]
If you know what the argument names (or documentation) are for any of those interfaces, let me know )

File: Scripts/Devices/helper_driver.groovy [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        dumpMethods(cls, result)
+    }
+
+    return result.join('\n')
+}
+
+def parse(String s)
+{
+    return [:]
+}
```
Comment (cmorse) [inline Scripts/Devices/helper_driver.groovy]: This is a creative way to get everything that's available. Much easier than scraping things from the smartthings or hubitat website.

Comment (biocomp) [reply Scripts/Devices/helper_driver.groovy]: Yeah, I gave up on looking at docs pretty early. Especially Hubitat's ones :)

**Pull request merged**